File: app/src/main/java/com/example/memory_jogger/MainActivity.java
package com.example.memory_jogger;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.quick_menu);

        Button playMenuButton = (Button) findViewById(R.id.normalButton);
        Button highScoreButton = (Button) findViewById(R.id.highScoreButton);

        playMenuButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                openActivityPlay();
            }
        });

        highScoreButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                openActivityHighScore();
            }
        });
    }

    public void openActivityPlay(){
        Intent intent = new Intent(this, playActivity.class);
        startActivity(intent);
    }

    public void openActivityHighScore() {
        Intent intent = new Intent(this, highScoreActivity.class);
        startActivity(intent);
    }
}

File: app/src/main/java/com/example/memory_jogger/MainActivity.kt
package com.example.memory_jogger

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*
import com.example.memory_jogger.playActivity2 as playActivity21

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {        //When class is created, do this{
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)                  //show this layout

        playMenuButton.setOnClickListener {             //when playMenuButton is clicked start playActivity
            startActivity(Intent(this, playActivity21::class.java))

        }
    }
}